mp4parse/src/lib.rs

```diff
@@ -1682,6 +1711,62 @@ fn read_stsd<T: Read>(src: &mut BMFFBox<T>, track: &mut Track) -> Result<SampleD
     })
 }
 
+fn read_sinf<T: Read>(src: &mut BMFFBox<T>) -> Result<ProtectionSchemeInfoBox> {
+    let mut sinf = ProtectionSchemeInfoBox::default();
+
+    let mut iter = src.box_iter();
+    while let Some(mut b) = iter.next_box()? {
+        match b.head.name {
+            BoxType::OriginalFormatBox => {
+                let frma = read_frma(&mut b)?;
+                sinf.code_name = frma;
+            },
```

Comment:
I don't have the `cenc` spec, but 14496-12 says the `OriginalFormatBox` is required. Can we reject streams without one here? We don't seem to check later whether this is what we expect.